sbin/common/config_init.sh

```diff
@@ -32,6 +32,8 @@
 # map. This is why we can't have nice things.
 CONFIG_PARAMS=(
 ADOPT_PATCHES
+ADOPTOPENJDK_BUILD_REPO_BRANCH
```

Comment:
I'm wondering, could/should we abstract this further so that it's not just Adopt, but any valid OpenJDK source repo?  So maybe it's 

`OPENJDK_BUILD_REPO_BRANCH`

Reply:
sure, I think your suggestion is better, will update.